hail/src/main/scala/is/hail/io/BufferSpecs.scala

```diff
@@ -12,9 +12,10 @@ import org.json4s.jackson.JsonMethods
 import org.json4s.{Extraction, JValue}
 
 object BufferSpec {
+  // FIXME (temporary?)
   val default: BufferSpec = LEB128BufferSpec(
-    BlockingBufferSpec(32 * 1024,
-      LZ4HCBlockBufferSpec(32 * 1024,
+    BlockingBufferSpec(64 * 1024,
+      ZstdBlockBufferSpec(64 * 1024,
         new StreamBlockBufferSpec)))
 
   val defaultUncompressed: BufferSpec = BlockingBufferSpec(32 * 1024,
```

Comment:
cc: @tpoterba 

Should defaultUncompressed now be default? [The sheet](https://docs.google.com/spreadsheets/d/1uDYfXWDIwt_-XiclFWdLDiXamVbxhswt_D-v8fVrlRU/edit#gid=0) indicates that for a 250MiB/s pipe (or smaller), compression with zstd is strictly an improvement. lz4 HC was 10x the time compared to uncompressed for writing, but zstd is 0.5x of the time.

Down below it looks like we have memorySpec and wireSpec. I'm not sure what memorySpec is. wireSpec should *definitely* use zstd.

Reply:
and yes, wireSpec should use Zstd for sure.

Reply:
I'm currently trying to iron out where the codebase is assuming lz4, and then I'll transition over to `wireSpec` being Zstd. And probably rename 'default' while I'm at it, since assuming 'default' has got us in trouble in the past and we should make ourselves be explicit.

Reply:
https://github.com/hail-is/hail/blob/92e4776161e732d6e6b18c0889a5c2901fc2f94a/hail/src/main/scala/is/hail/io/index/IndexWriter.scala#L78